app/workers/shipment_worker.rb

```diff
@@ -0,0 +1,15 @@
+class ShipmentWorker
+  include Sidekiq::Worker
+
+  def perform(*args)
+    five_days_later = 432000
```

Comment:
So the default shipment time will be 5 days from now. If this were actually being used I would have to schedule the job such that it runs at the beginning of the week, so always on Monday/Sunday night, that way the delivery date would always be Thursday. In a real world setting that would make the most sense for a small baking company